src/quo2/components/channel_avatar.cljs

```diff
@@ -0,0 +1,49 @@
+(ns quo2.components.channel-avatar
+  (:require [quo2.foundations.colors :as colors]
+            [quo.react-native :as rn]
+            [clojure.string :as clj-string]
+            [status-im.ui.components.icons.icons :as icons]))
+
+(defn channel-avatar [{:keys [big? dark? lock-status icon x y]}]
+  (let [locked? (= :locked lock-status)
+        lock-exists? (not= :none lock-status)]
+    [rn/view {:style {:width (if big? 32 24)
+                      :height (if big? 32 24)
+                      :top (if (or (clj-string/blank? x)
+                                   (js/isNaN x))
+                             0
+                             (js/parseInt x))
+                      :left (if (or (clj-string/blank? y)
```

Comment:
you could just use `int` without any conditions

Reply:
Done